nss-day-cohort-22/company-website-part-deux-lopsided-pelicans
# Rm testimonials

1. git fetch --all  to retrieve branch
1. git checkout rm-testimonials
1. cd testimonials 
1. start http-server
1. open browser to 127.0.0.1:8080/testimonials.html
1. verify that slideshow is working and moving automatically through 4 images
